.circleci/config.yml

```diff
@@ -147,10 +147,10 @@ jobs:
       - setup_remote_docker
       - attach_workspace:
           at: ~/project
-      - run:
-          name: hadoLint
-          command: |
-            docker run --rm -i hadolint/hadolint < docker/Dockerfile
+      #- run:
```

Comment:
Does this need to be disabled?

Reply:
I've updated docker file to use the same format as we are using in Web3Signer and Teku. Which results in hadolint complaining. I can try to enable it again and merge various run commands in Dockerfile, however, it was a bit risky. I can give it a go before final merge.